test/nuts/suggestType/suggestType.nut.ts

```diff
@@ -66,6 +69,38 @@ describe('suggest types', () => {
     }
   });
 
+  it('it offers a suggestions on metadata format file when converting to metadata', async () => {
+    const lifecycleSpy = sinon.spy(Lifecycle.prototype, 'emitWarning');
+
+    const set = await ComponentSetBuilder.build({
+      sourcepath: [
+        path.join(
+          session.project.dir,
+          'force-app',
+          'main',
+          'default',
+          'enablementMeasureDefinitions',
+          'measure.enablementMeasureDefinition'
+        ),
+      ],
+    });
+    const converter = new MetadataConverter();
+    await converter.convert(set, 'metadata', {
+      type: 'directory',
+      genUniqueDir: false,
+      outputDirectory: 'output',
+    });
+    expect(lifecycleSpy.calledOnce).to.be.true;
+    expect(lifecycleSpy.args[0][0]).to.equal(
```

Comment:
can you write this to assert that the expected error exists, not its ordinality, so that some future warning doesn't break the UT?